api/models/cartDao.js

```diff
@@ -0,0 +1,18 @@
+const { appDataSource } = require("./dataSource");
+
+const cartLookUpByUserId = async (userId) => {
+  const cartLookUpByUserId = await appDataSource.query(`
+    SELECT 
+    products.name productName,
+    product_options.name productOptionName,
+    carts.quantity quantity,
+    products.price*quantity price
+    FROM carts
+    JOIN products ON carts.product_id = products.id
+    JOIN product_options ON carts.product_id = product_options.product_id
+    WHERE carts.user_id = ${userId};
```

Comment:
- data를 줄 때는 id를 꼭 포함 시켜 주세요.
- 가격 계산은 frontend에서 해주면 됩니다.
- 
```suggestion
    SELECT 
      cart.id AS cartId
      products.name AS productName,
      product_options.name AS productOptionName,
      carts.quantity AS quantity,
      products.price AS price
    FROM carts
    JOIN products ON carts.product_id = products.id
    JOIN product_options ON carts.product_id = product_options.product_id
    WHERE carts.user_id = ?`, [userId];
```